core/include/aes_gcm.h

```diff
@@ -5,9 +5,11 @@
 // M_KeyID is typedef'ed in codesafe header. If STDMARSHALTYPES_H is not
 // defined, we are in dev target
 #ifndef STDMARSHALTYPES_H
-typedef enum {KEYID_MASTER_SEED_ENCRYPTION_KEY = 0,
-              KEYID_PUBKEY_ENCRYPTION_KEY = 1} M_KeyID;
+typedef enum {master_seed_encryption_key = 1,
```

Comment:
It's possible to initialize the KEK values elsewhere. However, because M_KeyID are handles to keys (for both targets) and the `0` is an invalid value in the ncipher target, it seems to me that the most portable approach is also treating `0` as an invalid handle value for the dev target. Therefore, I'd like to argue that it's not intrinsic to change the location of KEK initialization, and perhaps better to have them declared in this header file, for simplicity.